README.md

```diff
@@ -26,6 +26,8 @@ gems:
 
 ## Configuration
 
+The configuration file is `_config.yml`, the same as Jekyll config file.
+
```

Comment:
Jekyll assets doesn't have a configuration file. It uses Jekyll configuration file because it is a plugin. Which is why it cannot be the same as... Because it is.